app/configurator/components/layout-configurator.tsx

```diff
@@ -290,8 +306,9 @@ const SharedFilterOptions = ({
   if (sharedFilter.type !== "timeRange") {
     return null;
   }
+
   if (
-    !isTemporalDimension(dimension) ||
+    !(isTemporalDimension(dimension) || isTemporalEntityDimension(dimension)) ||
```

Comment:
I guess this check and the ones before could be refactored as a "canDimensionBeTimeFiltered" or something like this ?